extensions/vscode/webviews/homeView/src/HostConduitService.test.ts

```diff
@@ -0,0 +1,80 @@
+import { setActivePinia, createPinia } from "pinia";
+import { beforeEach, describe, expect, test, vi } from "vitest";
+import { App, createApp } from "vue";
+
+import { HostToWebviewMessageType } from "../../../src/types/messages/hostToWebviewMessages";
+import { WebviewToHostMessageType } from "../../../src/types/messages/webviewToHostMessages";
+import { useHostConduitService } from "src/HostConduitService";
+import { useHomeStore } from "src/stores/home";
+import { vscodeAPI } from "src/vscode";
+import { WebviewApi } from "vscode-webview";
+
+export function withSetup<T>(composable: () => T): App {
```

Comment:
We can probably break this out into another util tests file at some point, but this helper created a simple App that used the composable so `onMount()` and similar lifecycle functions worked for composables.